src/components/App/App.js

```diff
@@ -1,27 +1,35 @@
 import React, { Component } from 'react';
 import './App.css';
-import movieData from '../../movieData';
+// import movieData from '../../movieData';
 import MoviesContainer from '../MoviesContainer/MoviesContainer';
 import MovieDetails from '../MovieDetails/MovieDetails';
+import { fetchMovies } from '../../apiCalls';
+import { fetchMovieInfo } from '../../apiCalls';
 
 class App extends Component {
   constructor() {
     super();
     this.state = {
-      movies: movieData.movies,
-      // isClicked: false,
+      movies: [],
       clickedMovie: {},
-      // error: ''
+      error: ''
     }
   }
 
-  updateIsClicked = () => {
-  this.setState({clickedMovie: {}})
+  componentDidMount = () => {
+    fetchMovies()
```

Comment:
This looks great for fetching single movie data. We should research what to do with the error parameter since it's potentially not being utilized? (not showing up as highlighted in our code)